.lintstagedrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   '*.{ts,tsx,vue,css,less,scss,html,htm,md,markdown}': 'prettier --write',
   '*.{js,jsx,json,yml,yaml}': ['prettier --write', () => 'npm run test'],
+  '*.{js,jsx}': ['eslint --fix'],
```

Comment:
```suggestion
  '*.{json,yml,yaml}': ['prettier --write', () => 'npm run test'],
  '*.{js,jsx}': ['eslint --fix', () => 'npm run test']],
```

ESLint will also run the prettier fixes via the plugin (or extend).

Reply:
Nice catch! I completely forgot that now the prettier plugin applies all prettier changes to the JS files